mars/learn/decomposition/tests/test_pca.py

```diff
@@ -39,6 +40,11 @@ def setUp(self):
         # solver_list not includes arpack
         self.solver_list = ['full', 'randomized', 'auto']
         self.session = new_session().as_default()
+        self.executor = Executor('numpy')
```

Comment:
We should not use `numpy` executor as a workaround.

Reply:
Sorry, I forget to update this pr